js/components/LoadingBar.jsx

```diff
@@ -0,0 +1,23 @@
+const React = require('react');
+
+const LoadingBar = function(props) {
+
+	if (!props.show) {
+		return null
+	}
+
+	return (
+		<div className="loading-horisontal">
+			<div id="loading-horisontal-1"></div>
```

Comment:
annoyingly inconsistent language as usual: `apologized` and `apologised` are valid, but `horisontal` is not!